### ISSUE TITLE
Move “Compose” up in sidebar

### ISSUE THREAD

**Issue by creohn** [elk-zone/elk#1086]
The sidebar actions should be ordered by priority, & since “Compose” is the key feature, it should be up in the top area 
<img alt="wireframe reorder sidebar" src="https://user-images.githubusercontent.com/9348528/212367494-b344264e-4135-47a6-afc0-94f6da611084.png">

**Comment by patak-dev** [comment]
We were discussing renaming "Compose" to "Composer", because here you'll also have access to your drafts (maybe media that isn't yet sent later?). And then add a new Compose icon button using primary color (orange) next to the back button on the top (and make these buttons more prominent, filled).

Does that sounds good for you?

**Comment by creohn** [comment]
Yes, good idea. & if you start using the top-area for action-icons, reduce “Home” to just the house-icon & also move it in the same line, since the elk logo has the go-home function/link anyways? 
<img width="300" alt="01_elk_home" src="https://user-images.githubusercontent.com/9348528/212376564-2ce0bc44-4082-4b06-88a3-5354be38dab9.png">

**Comment by forenta** [comment]
> And then add a new Compose icon button using primary color (orange) next to the back button on the top (and make these buttons more prominent, filled).

I'm not entirely sure how the majority hold their phone when doomscrolling - but for what reason could a positioning other than *right at the bottom* make sense? Is this handled differently by the big platforms somewhere?

**Comment by kaiserkiwi** [comment]
Oh wow. I just realized that there is a compose button because of this issue. 
I would definitely suggest to make it visually distinct too.
Something like that. It's hard to come up with a good design because of the "clean" look of Elk without contrasts etc. But I think the most important action should stand out.
![image](https://user-images.githubusercontent.com/8428551/221877276-95da3615-c642-4917-9f85-5c7306b9f243.png)
